symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/datafeed/DatafeedLoop.java

```diff
@@ -37,4 +37,11 @@ public interface DatafeedLoop {
      * @param listener a Datafeed event listener to be unsubscribed
      */
     void unsubscribe(RealTimeEventListener listener);
+
+  /**
+   * The timestamp of the last successful pulling
+   *
+   * @return timestamp value in long
+   */
+  long lastPullTimestamp();
```

Comment:
I don't think we want to expose this method in DF loop api to users.

Reply:
then how to get this timestamp in a generic way ? another way is to move the health check method inside the DatafeedLoop,  but i would say expose a last pull timestamp method in this interface is more relevant than exposing a health check method.